code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -2015,6 +2015,25 @@
 							src.show_text(__red("Your head hurts!"))
 				else
 					src.show_text("You don't know how to do that but you feel deeply ashamed for trying", "red")
+			if("carnivalcant")
+				if((src.job == "Clown") || (src.job == "Mime"))
+					var/list/gibberish = list("looks like [he_or_she(src)] is trying to speak a really awkward sign language...",
+					"looks as if [he_or_she(src)] is trying to communicate with [his_or_her(src)] fingers...",
+					"twiddles [his_or_her(src)] fingers strangely...", "forms interesting symbols with [his_or_her(src)] fingers...",
+					"flaps [his_or_her(src)] hands around like a butterfly...")
+					SPAWN_DBG(0)
+						var/old_x = src.pixel_x
+						var/old_y = src.pixel_y
+						src.pixel_x += rand(-3,3)
+						src.pixel_y += rand(-1,1)
+						sleep(0.2 SECONDS)
+						src.pixel_x = old_x
+						src.pixel_y = old_y
+					var/chosen_gibberish = pick(gibberish)
+					maptext_out = chosen_gibberish
```

Comment:
need to set a message as well if you want overhead text, otherwise this will only do the minor pixel shift and nothing else
```suggestion
					maptext_out = "<i>[chosen_gibberish]</i>"
					message = "<b>[src]</b> [chosen_gibberish]"
```

Reply:
there is a message, but i *did* forget to italicise it :P

Reply:
When I tried it locally it didn't work at all until I added that message var

Reply:
hmmm....eek you're right...